### DISCUSSION TITLE
Genesys AudioHook Integration -- Custom JSON

### DISCUSSION THREAD

**Discussion by JamesKiddSmyth** [deepgram/community#703] [General help] [answered]
I've written an Genesys AudioHook emulator in order to test an AudioHook server-side implementation.  I would like to test more against (and do some comparison with) the [Deepgram Genesys integration](https://developers.deepgram.com/docs/genesys-with-deepgram).  In there, it says "Under Configuration > Advanced, provide a JSON object to customize the Deepgram request. For example:". **Can someone tell me where that custom json would go in the AudioHook messaging?**  I've tried putting in a few places in the "open" message with no luck.  I do get all the proper responses from the Deepgram end of things (ie. "opened", "closed", "pong"), but I'm not getting my callback hit.

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by RyanChimienti** [comment, marked as answer]
Hi @JamesKiddSmyth, that object gets sent in the `customConfig` field of the open message, documented [here](https://developer.genesys.cloud/devapps/audiohook/protocol-reference#openparameters). Something like this should work:

```
customConfig: {
  "callback": "https://yourserver.com"
}
```

**Reply by JamesKiddSmyth** [reply]
Yes! This works perfectly.  Thank you.
